website/src/theme/DocCategoryGeneratedIndexPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {useLayoutDoc} from '@docusaurus/theme-common';
+import Link from '@docusaurus/Link';
+import Translate from '@docusaurus/Translate';
+import DocCategoryGeneratedIndexPage from '@theme-original/DocCategoryGeneratedIndexPage';
+import type {Props} from '@theme/DocCategoryGeneratedIndexPage';
+
+import styles from './styles.module.css';
+
+function HintFooter() {
+  const docPath = useLayoutDoc('guides/docs/sidebar/items', undefined)?.path;
```

Comment:
Ref https://github.com/facebook/docusaurus/issues/7402 a `DocLink` component will make this a bit easier